frontend/tests/features/wheresMyGene/downloadCsv.test.ts

```diff
@@ -28,7 +28,13 @@ describe("CSV download tests", () => {
     //download  csv file
     await downloadAndVerifyFiles(page, fileTypes, tissues, folder);
     // verify csv file
-    await verifyCsv(page, folder, "disease-filter", SHARED_LINK_NO_GROUP);
+    await verifyCsv({
```

Comment:
verifyCSV needs to be updated to take tissue aggregated rows in CSV into account to be accurate again